src/test/java/com/sap/sgs/phosphor/fosstars/data/github/UsesGithubForDevelopmentTest.java

```diff
@@ -130,4 +132,28 @@ public void testWhenGettingRepositoryFails() throws IOException {
     assertTrue(values.of(USES_GITHUB_FOR_DEVELOPMENT).isPresent());
     assertTrue(values.of(USES_GITHUB_FOR_DEVELOPMENT).get().isUnknown());
   }
+
+  @Test
+  public void testWithGithubDirectory() throws IOException {
```

Comment:
Could you please add a negative test case as well. To check.
```
catch (IOException e) {
  return false;
}
...
```
And confirm `assertEquals(Boolean.FALSE, values.of(USES_GITHUB_FOR_DEVELOPMENT).get().get());`

Reply:
Sure, I've added a test.